src/SonarQube.Client/Requests/RequestFactory.cs

```diff
@@ -61,7 +61,7 @@ public RequestFactory(ILogger logger)
             return RegisterRequest<TRequest, TRequestImpl>(version, () => new TRequestImpl());
         }
 
-        private RequestFactory RegisterRequest<TRequest, TRequestImpl>(string version, Func<TRequestImpl> factory)
+        public RequestFactory RegisterRequest<TRequest, TRequestImpl>(string version, Func<TRequestImpl> factory)
```

Comment:
Was this intentional? It's only used inside the file

Reply:
Good catch! I was experimenting with using factory instead of new() constraint for generics and forgot to remove it